dali/operators/bbox/bbox_paste.cc

```diff
@@ -20,13 +20,14 @@ namespace dali {
 
 DALI_SCHEMA(BBoxPaste)
     .DocStr(
-        R"code(Transforms bounding boxes so that they are in the same place in the image after pasting it onto a larger canvas.
+        R"code(This operator transforms bounding boxes so that the boxes  remain in the same
+place in the image after the image is pasted on a larger canvas.
 
-Corner coordinates::
+Here are the corner coordinates::
```

Comment:
A general remark: I don't think this kind of header is common in documentations - so far, I've rather encountered the contrary. Also, it's inconsistent with headers generated by Sphinx, which look like:
``Parameters``
``Returns``
etc

Reply:
No opinion, to be honest.

Reply:
So what is the conclusion?